static/js/components/SourceCounts.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { CountUp } from "use-count-up";
+
+import Paper from "@material-ui/core/Paper";
+import Typography from "@material-ui/core/Typography";
+
+import WidgetPrefsDialog from "./WidgetPrefsDialog";
+import * as profileActions from "../ducks/profile";
+
+import styles from "./TopSources.css";
+
+const defaultPrefs = {
+  sinceDaysAgo: "",
+};
+
+const SourceCounts = () => {
+  const sourceCounts = useSelector((state) => state.sourceCounts.sourceCounts);
+  const sourceCountPrefs =
+    useSelector((state) => state.profile.preferences.sourceCounts) ||
+    defaultPrefs;
+
+  return (
+    <Paper elevation={1}>
+      <div className={styles.topSourcesContainer}>
+        <div style={{ display: "inline-block", float: "right" }}>
+          <WidgetPrefsDialog
+            formValues={sourceCountPrefs}
+            stateBranchName="sourceCounts"
+            title="Source Count Preferences"
+            onSubmit={profileActions.updateUserPreferences}
+          />
+        </div>
+        <div style={{ display: "inline-block", align: "center" }}>
```

Comment:
We use non-inline styles now (please see the style guide for how to add style classes).